subprojects/docs/src/snippets/testing/jacoco-quickstart/groovy/build.gradle

```diff
@@ -33,10 +33,10 @@ test {
 // end::testtask-configuration[]
 
 // tag::testtask-dependency[]
-test {
-    finalizedBy jacocoTestReport // report is always generated after tests run
+tasks.named("check").configure {
```

Comment:
These changes make no sense in isolation. The text before the inclusion of that snippet should be adapted as well then.

Another option is to make it clearer here that what is proposed is _optional_